le_utils/constants/file_formats.py

```diff
@@ -44,6 +44,10 @@
 HTML5 = "zip"
 HTML5_MIMETYPE = ".zip"
 
+# constants for ePub format
+EPUB = "epub"
+EPUB_MIMETYPE = "application/epub+zip"
```

Comment:
Need to check on Studio if dropzone allows this (otherwise, needs to be '.epub'). Also, if .epub3 is supported in Kolibri, can add that here